### PR TITLE
Rl header

### PR DESCRIPTION
**Pull request by Kurkos**
Completed Header component.
CSS needs to be fixed.

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+import {NavLink} from 'react-router-dom';
+import './Header.css';
+
+
```
Comment (janisknets) [inline src/components/Header/Header.js]: unneeded newline

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+class Header extends Component {
+
+  render() {
+    return (
+
```
Comment (janisknets) [inline src/components/Header/Header.js]: also unneded white newline

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+import {NavLink} from 'react-router-dom';
+import './Header.css';
+
+
+class Header extends Component {
+
```
Comment (janisknets) [inline src/components/Header/Header.js]: unneded newline

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            <div>
+                <h3 className="Header__title">Brand Name</h3>
+            </div>
+            <div className="Header__links">
+                <NavLink className="Header__link" to="/"  activeClassName="selected" >QUIZZES</NavLink>
+                <NavLink className="Header__link" to="/results" >RESULTS</NavLink>
+                <NavLink className="Header__link" to="/LogOut">LOG OUTS</NavLink>
+            </div>
+        </div>
+
```
Comment (janisknets) [inline src/components/Header/Header.js]: unneded newline

File: src/App.js [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
+
 
 class App extends Component {
   render() {
-    return <div className="App">
-    <Login/>
-    </div>;
+    return (
+      <div className="App">React App</div>
+
```
Comment (janisknets) [inline src/App.js:11]: newline

File: src/App.js [hunk 6]
```diff
@@ -1,12 +1,13 @@
 import React, { Component } from "react";
 import "./App.css";
-import Login  from  "./components/Login/Login"
+
```
Comment (janisknets) [inline src/App.js:4]: newline

**Review by janisknets — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.css [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1;
+
+    display: flex;
+    justify-content: space-between;
+
+    background: white;
+    height:60px;
```
Comment (rokija) [inline src/components/Header/Header.css]: add space between height: and 60px

**Review by rokija — COMMENTED** [review]
put it somewhere visible in the page please, for example in App.js

File: src/components/Header/Header.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import {NavLink} from 'react-router-dom';
```
Comment (janisknets) [inline src/components/Header/Header.js]: add spaces before and after NavLink

**Review by janisknets — APPROVED** [review]

**Comment by rokija** [comment]
screenshot 2019-02-19 at 09 40 37

**Review by rokija — COMMENTED** [review]
When a page is selected in the navigation I don't see which one it is (active className is not working)

**Review by rokija — APPROVED** [review]

**Pull request merged**